swagger.yaml

```diff
@@ -996,6 +996,55 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/deviceData/{deviceId}/surveys/{deviceSurveyId}/pages/{pageId}/answers:
+    post:
+      operationId: Submit survey answer
+      summary: Submits page answer for the survey
+      description: Endpoint for the devices to send their responses to the survey pages
+      tags:
+        - DeviceData
+      parameters:
+        - name: deviceId
+          required: true
+          in: path
+          description: Device id
+          schema:
+            type: string
+            format: uuid
+        - name: deviceSurveyId
+          required: true
+          in: path
+          description: device survey id
+          schema:
+            type: string
+            format: uuid
+        - name: pageId
+          required: true
+          in: path
+          description: Device id
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Layout"
+        description: Payload
+        required: true
+      responses:
+        "200":
+          description: Created Layout
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Layout"
```

Comment:
Remove this if you use http 202, or change to DevicePageSurveyAnswer when using 200